examples/compiled/concat_layer_voyager_result_future.vg.json

```diff
@@ -70,16 +70,6 @@
       "name": "data_3",
       "source": "data_1",
       "transform": [{"type": "filter", "expr": "datum.label !== 'PoleStar'"}]
-    },
```

Comment:
In the future PRs, we should try to see how we can make data_2 and data_3 merge for this example.  (No need for this PR.)